### PR TITLE
adjust workspace directory

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
Not 100% sure when this changed, but I think with the Jenkins re-setup. Now only the files generated with Puppet are in `/var/lib/workspace` and this PR would change it to the current Jenkins default.

File: puppet/modules/deploy/manifests/slave.pp [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class deploy::slave {
```
Comment (ekohl) [inline puppet/modules/deploy/manifests/slave.pp]: This class is unused now. 767f64c23d111570bda8de9b44b5e645c12cbf65 removes it.

File: puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ DEB_PATH=./debian/${os}/build-${project}
 
 # Upload all builds to stagingdeb for testing
 echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
-export RSYNC_RSH="ssh -i /var/lib/workspace/workspace/staging_key/rsync_freightstage_key"
+export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
```
Comment (ekohl) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/debian/stage_packages.sh]: Ideally we'd use Jenkins secrets similar to 066cfae0ea07d5c15fc2488853971c8f609f21e8 and ad703c92d5dd12599c044ff438c4d0aefe9b24bb for this.

File: puppet/modules/web/manifests/uploader.pp [hunk 3]
```diff
@@ -5,7 +5,7 @@
```
Comment (ekohl) [inline puppet/modules/web/manifests/uploader.pp]: Unused and 187db02435bddc3b49b3ab37ea39f438a3f70b14 drops it.

**Comment by ekohl** [comment]
This now has merge conflicts. What do you want to do with this?

**Comment by mmoll** [comment]
resolving the conflicts, of course :laughing:

**Comment by mmoll** [comment]
I cleaned out the old directory with _rex. Let's see, if anything fails now :bomb:.

**Pull request merged**